lib/bloc/account/account_model.dart

```diff
@@ -13,35 +14,41 @@ class AccountSettings {
   final bool ignoreWalletBalance;
   final bool showConnectProgress;
   final BugReportBehavior failedPaymentBehavior;
+  final bool isEscherEnabled;
 
   AccountSettings(this.ignoreWalletBalance,
       {this.showConnectProgress = false,
-      this.failedPaymentBehavior = BugReportBehavior.PROMPT});
+      this.failedPaymentBehavior = BugReportBehavior.PROMPT,
+      this.isEscherEnabled});
 
   AccountSettings.start() : this(false);
 
   AccountSettings copyWith(
       {bool ignoreWalletBalance,
       bool showConnectProgress,
-      BugReportBehavior failedPaymentBehavior}) {
+      BugReportBehavior failedPaymentBehavior,
+      bool isEscherEnabled}) {
     return AccountSettings(ignoreWalletBalance ?? this.ignoreWalletBalance,
         showConnectProgress: showConnectProgress ?? this.showConnectProgress,
         failedPaymentBehavior:
-            failedPaymentBehavior ?? this.failedPaymentBehavior);
+            failedPaymentBehavior ?? this.failedPaymentBehavior,
+        isEscherEnabled: isEscherEnabled ?? this.isEscherEnabled);
   }
 
   // typo isn't fixed on json to prevent unexpected behavior
   AccountSettings.fromJson(Map<String, dynamic> json)
       : this(json["ignoreWalletBalance"] ?? false,
             showConnectProgress: json["showConnectProgress"] ?? false,
             failedPaymentBehavior:
-                BugReportBehavior.values[json["failePaymentBehavior"] ?? 0]);
+                BugReportBehavior.values[json["failePaymentBehavior"] ?? 0],
+            isEscherEnabled: json["isEscherEnabled"] ?? Platform.isIOS);
```

Comment:
Let's leave this false by default to all platform. For now it can only be enabled at the development console.

Reply:
[Done](https://github.com/breez/breezmobile/pull/336/commits/d09d109a0c8e0250c885bbd64c1e39d70a9f02b2)